docs/api/wrapper/emitted.md

```diff
@@ -9,26 +9,30 @@ Return an object containing custom events emitted by the `Wrapper` `vm`.
 ```js
 import { mount } from '@vue/test-utils'
 
-const wrapper = mount(Component)
+test('emit demo', async () => {
+  const wrapper = mount(Component)
 
-wrapper.vm.$emit('foo')
-wrapper.vm.$emit('foo', 123)
+  wrapper.vm.$emit('foo')
+  wrapper.vm.$emit('foo', 123)
 
-/*
-wrapper.emitted() returns the following object:
-{
-  foo: [[], [123]]
-}
-*/
+  await wrapper.vm.$nextTick() // Wait until $emits have been handled
```

Comment:
Not sure if it's too down-to-metal, but we're technically waiting for the next DOM update cycle, right? Should be mention that? I mean, `$nextTick` has nothing to do with `$emits` directly, and one could infer that from the comment above.

Just thinking out loud 😇

Reply:
What you say is true, although I actually think the update @samgre made is fine. I think the docs should just show you how to use the library, rather than explain the implementation details.

If you want to put another commit on top of this making the comment more accurate, that's fine too. Perhaps "await until the next cycle to ensure $emits have been handle" might be appropriate.